### ISSUE TITLE
Favicons are not visible prominently in Dark mode

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#12810]
## Description 

favicons are not visible prominently in Dark mode

## Steps to Reproduce

   1. Open Brave 1.18.x
   2. Change appearance settings to Dark mode
   3. Visit brave://settings/cookies
   4. Open a PT window and visit any site
   5. Come back to brave://settings/cookies again
   6. favicons are not visible prominently in Dark mode
   

## Actual result:

favicons are not visible prominently in Dark mode
![image](https://user-images.githubusercontent.com/38657976/99805025-e6dd0a80-2b61-11eb-835a-f87e6c03d036.png)

## Expected result:
Favicons should be lighter color in dark theme

## Reproduces how often: 

Always

## Brave version (brave://version info)

Brave | 1.18.57 Chromium: 87.0.4280.66 (Official Build) beta (64-bit)
-- | --
Revision | fd98a29dd59b36f71e4741332c9ad5bda42094bf-refs/branch-heads/4280@{#1432}
OS | Windows 10 OS Version 2004 (Build 19041.572)

## Version/Channel Information:

- Can you reproduce this issue with the current release? No
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

## Miscellaneous Information:

cc: @brave/legacy_qa @rebron @karenkliu

**Comment by karenkliu** [comment]
Yeah just make it the same color as the globe icon to its left. Not sure the theme isn't picking it up.

**Comment by PreetibarnaPanda** [comment]
Make the colours of the favicons save as the ones in the settings so as they they stand out in the particular screen

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
